test.ml

```diff
@@ -10,6 +10,6 @@ let () =
     assert (s = Lzo.decompress (Lzo.compress s) (String.length s));
```

Comment:
The tests are executed in the library. You should separate them into two directories and add a proper `dune` file for the tests